src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 12px;
+  width: 12px;
+  margin: 2px 4px;
+  background-size: cover;
+}
+
+.stars__star:first-child {
+  margin-left: 2px;
+}
+
+.stars :nth-child(-n+4) {
```

Comment:
You need to be able to set active stars with a modifier in HTML. This CSS code doesn't seem to support that.

Reply:
Change done